hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/FilterParams.java

```diff
@@ -0,0 +1,178 @@
+/**
+ * Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.repository;
+
+import java.util.Collection;
+
+import org.eclipse.hawkbit.repository.model.DistributionSet;
+import org.eclipse.hawkbit.repository.model.TargetUpdateStatus;
+
+/**
+ * Encapsulates a set of filters that may be specified (optionally). Properties
+ * that are not specified (e.g. <code>null</code> for simple properties) When
+ * applied, these filters are AND-gated.
+ *
+ */
+public class FilterParams {
+
+    Collection<TargetUpdateStatus> filterByStatus;
```

Comment:
make use of the `private` modified for the fields in `FiterParams`, so fields get not accessed directly.